libs/openFrameworks/gl/ofGLProgrammableRenderer.cpp

```diff
@@ -1315,12 +1321,46 @@ void ofGLProgrammableRenderer::bind(const ofFbo & fbo, bool setupPerspective){
 }
 
 //----------------------------------------------------------
-void ofGLProgrammableRenderer::unbind(const ofFbo & fbo){
-	glBindFramebuffer(GL_FRAMEBUFFER, 0);
+void ofGLProgrammableRenderer::end(const ofFbo & fbo){
+	fbo.unbind();
```

Comment:
> In such a scenario it appears FBOs would not be shareable, because they are Container objects, like VAOs (https://www.opengl.org/wiki/OpenGL_Object#Container_objects).

also this is why now every window contains it's own renderer, some resources can't be shared so the programmable renderer wouldn't work if there was only one renderer for all the windows as it was till now, the shaders won't work across different contexts even if they are shared.

but this is not something we should care about for all the objects, if someone wants to use openGL in such an advanced scenario i think it's safe to suppose that they'll now which objects can be shared or not